h2o-py/h2o/group_by.py

```diff
@@ -50,52 +73,143 @@ def __init__(self, fr, by):
+            get_frame().
+        """
         return self._add_agg("sdev", col, na)
 
 
     def var(self, col=None, na="all"):
+        """
+        Calculate the variance (per numerical column) in each group of a GroupBy object for column(s) specified by
+        the user.
+
+        :param str/int/list_of_str/list_of_int col: representing column name(s) or column index(indices) that you want
+            to perform aggregation on.  If no col is given to the aggregation, the aggregation would be applied to
+            all columns but the group by columns.
+        :param str na:  one of 'rm', 'ignore' or 'all' (default).
+        :return: aggregation result stored as member _res of the GroupBy object and can be accessed by method
+            get_frame().
+        """
         return self._add_agg("var", col, na)
 
 
-    # def first(self,col=None,na="all"): return self._add_agg("first",col,na)
-    # def last( self,col=None,na="all"): return self._add_agg("last",col,na)
     def ss(self, col=None, na="all"):
+        """
+        Calculate the sum of squares (per numerical column) in each group of a GroupBy object for column(s) specified
+        by the user.
+
+        :param str/int/list_of_str/list_of_int col: representing column name(s) or column index(indices) that you want
+            to perform aggregation on.  If no col is given to the aggregation, the aggregation would be applied to
+            all columns but the group by columns.
+        :param str na:  one of 'rm', 'ignore' or 'all' (default).
+        :return: aggregation result stored as member _res of the GroupBy object and can be accessed by method
+            get_frame().
+        """
         return self._add_agg("sumSquares", col, na)
 
 
     def mode(self, col=None, na="all"):
+        """
+        Calculate the mode (per categorical column) in each group of a GroupBy object for column(s) specified by the
+        user.  Note that this metric is only meaningful categorical columns.
+
+        :param str/int/list_of_str/list_of_int col: representing column name(s) or column index(indices) that you want
+            to perform aggregation on.  If no col is given to the aggregation, the aggregation would be applied to
+            all columns but the group by columns.
+        :param str na:  one of 'rm', 'ignore' or 'all' (default).
+        :return: aggregation result stored as member _res of the GroupBy object and is accessed by method get_frame().
+        """
         return self._add_agg("mode", col, na)
 
 
     @property
     def frame(self):
-        """The resulting frame of the group by."""
+        """Return the resulting H2OFrame containing the result(s) of aggregation(s) of the group by."""
         return self.get_frame()
 
 
     def get_frame(self):
-        """The resulting frame of the group by."""
-        if not self._res:
+        """Return the resulting H2OFrame containing the result(s) of aggregation(s) of the group by."""
```

Comment:
More details needed here: what will be the structure of the resulting frame? What columns will it have?